CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
```

Comment:
```suggestion
## [0.6.2] - unreleased
```

We can release this as a patch release. Please also bump the version in `Cargo.toml`.